github-issues.go

```diff
@@ -195,6 +199,10 @@ func populateIssues(url string) {
 			eachGitIssue.Title = elem.Title
 			eachGitIssue.Repo = elem.RepositoryURL
 			eachGitIssue.Link = elem.HTMLURL
+			eachGitIssue.CreatedAt = elem.CreatedAt.Format(layout)
+			eachGitIssue.UpdatedAt = elem.UpdatedAt.Format(layout)
+			delta := elem.UpdatedAt.Sub(elem.CreatedAt)
+			eachGitIssue.Age = (int)(delta.Hours())
```

Comment:
Ok how about this. I will change the casting and use int64 in go style. But since i want only Duration's hours field i will continue to keep it as int64 https://golang.org/pkg/time/#Duration.Hours